userProfile/src/components/custom-modal/custom-modal.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Button, Modal } from '@skbkontur/react-ui';
+
+type CustomModalProps = {
+    message: string[];
+    onClose: () => void;
+};

```

Comment:
Теперь можно принимать в себя состояния формы
```suggestion
type CustomModalProps = {
    onClose: () => void;
    formData: {
        prevState: null | FormState;
        state: FormState;
    };
};
```